src/main/java/com/zufar/icedlatte/security/api/UserAuthenticationService.java

```diff
@@ -24,11 +25,13 @@ public class UserAuthenticationService {
     private final AuthenticationManager authenticationManager;
     private final LoginFailureHandler loginFailureHandler;
     private final ResetLoginAttemptsService resetLoginAttemptsService;
+    private final TokenManager tokenManager;
+
+    int userAccountLockoutDurationMinutes = 30;
```

Comment:
do we need to restrict the visibility to the private?

Reply:
Fixed